src/app/containers/StoryPromo/Figure.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { storyItemImage } from '../../models/propTypes/storyItem';
+import Figure from '../Figure';
+
+const StoryPromoFigure = ({ path, altText, height, width }) => {
+  if (!path || !altText || !width || !height) {
+    return null;
+  }
+
+  const ratio = ((height / width) * 100).toFixed(4);
+
+  return (
+    <Figure
+      alt={altText}
+      ratio={ratio}
+      src={`https://ichef.bbci.co.uk/news/660${path}`}
```

Comment:
Should this vary based on environment?

Reply:
Not yet, it matches [the current articles url](https://github.com/bbc/simorgh/blob/latest/src/app/containers/Image/index.jsx#L27), when we properly do image handling we can extract the url building :)